nmostesting/suites/IS0501Test.py

```diff
@@ -592,111 +592,126 @@ def test_25(self, test):
         """Immediate activation of a sender is possible"""
 
         if len(self.senders) > 0:
+            warn = ""
             for sender in self.is05_utils.sampled_list(self.senders):
                 valid, response = self.is05_utils.check_activation("sender", sender,
                                                                    self.is05_utils.check_perform_immediate_activation,
                                                                    self.transport_types[sender],
                                                                    True)
                 if valid:
+                    if response and not warn:
+                        warn = response
                     if self.transport_types[sender] == "urn:x-nmos:transport:rtp":
                         valid2, response2 = self.is05_utils.check_sdp_matches_params(sender)
                         if not valid2:
                             return test.FAIL("SDP file for Sender {} does not match the transport_params: {}"
                                              .format(sender, response2))
                 else:
                     return test.FAIL(response)
-            return test.PASS()
+            return test.PASS(warn)

```

Comment:
I am not too familiar with the `GenericTest` class, does this result in a warning when it takes more than 1 retry?

Reply:
No, it'll result in a (green) PASS but with the message in the details.
Now that the test pins down the sleep time with the leeway for slightly unsynchronization between testing tool and device under test, I agree we could make this a WARNING... @andrewbonney ?

Reply:
Yes I agree this could be a warning. It may just benefit from some extra explanatory text in addition.